tests/ms2/process-list/process-list.spec.ts

```diff
@@ -543,3 +546,296 @@ test('create a new folder and process, move process to folder and then delete bo
+    await page.getByRole('main').click();
+
+    /* Select all */
+    await page.locator('body').press('Control+a');
+
+    /* Check if only XYZ is selected */
+    await expect(page.getByRole('note')).toContainText('1');
+
+    /*  Search for A */
+    await inputSearch.getByPlaceholder(/search/i).fill('A');
+
+    /* Unfocus search */
+    await page.getByRole('main').click();
+
+    /* Select all */
+    await page.locator('body').press('Meta+a');
+
+    /* Check if both AAA are selected */
+    await expect(page.getByRole('note')).toContainText('2');
+  });
+
+  /* TODO: */
+  // test('select all of a specific page', async ({ processListPage }) => {});
+
+  // test('Select multiple with ctrl / meta and click', async ({ processListPage }) => {});
+
+  // test('Drag select with shift + click', async ({ processListPage }) => {});
+
+  /* Copy and Paste Processes - ctrl / meta + c -> ctrl / meta + v */
+  test('copy and paste processes with ctrl + c -> ctrl + v', async ({ processListPage }) => {
+    const { page } = processListPage;
+
+    /* Create a process */
+    const processID = await processListPage.createProcess({
+      processName: 'Copy me via shortcut',
+      returnToProcessList: true,
+    });
+
+    /* Select process */
+    await page.locator(`input[name="${processID}"]`).click();
+
+    /* Copy & Paste*/
+    await page.getByRole('main').press('Control+c');
+    await page.getByRole('main').press('Control+v');
+
+    await page.waitForTimeout(1_000); /* Ensure that animation is over */
+
+    /* Check if Modal is visible */
+    const modal = await page.getByRole('dialog');
+    await expect(modal, 'Could not open export modal with shortcut').toBeVisible();
+
+    /* Check if correct modal opened */
+    const modalTitle = await modal.locator('div[class="ant-modal-title"]');
+    await expect(modalTitle, 'Could not ensure that the correct modal opened').toHaveText(/copy/i);
+
+    /* Submit copy */
+    await page.getByRole('main').press('Control+Enter');
+
+    await page.waitForTimeout(1_000); /* Ensure that animation is over */
+
+    /* Check if Process has been added */
+    await expect(
+      page.locator('tbody>tr'),
+      'Could not find copied process in Process-List',
+    ).toHaveCount(2);
+
+    /* Copy & Paste - META */
+    await page.getByRole('main').press('Meta+c');
+    await page.getByRole('main').press('Meta+v');
+
+    /* Check if Modal is visible */
+    const modal2 = await page.getByRole('dialog');
+    await expect(modal, 'Could not open copy modal with shortcut (meta)').toBeVisible();
+
+    /* Check if correct modal opened */
+    const modalTitle2 = await modal2.locator('div[class="ant-modal-title"]');
+    await expect(modalTitle2, 'Could not ensure that the correct modal opened').toHaveText(/copy/i);
+
+    /* Submit copy */
+    await page.getByRole('main').press('Meta+Enter');
+
+    /* Check if Process has been added */
+    await expect(page.locator('tbody>tr')).toHaveCount(3);
```

Comment:
We should maybe be more specific in the future and actually check the names, like a user would read them.

Reply:
Added name check, left the count check nevertheless